router/pkg/config/config.go

```diff
@@ -946,6 +965,40 @@ func LoadConfig(configFilePath string, envOverride string) (*LoadResult, error)
 		}
 	}
 
+	// Merge in defaults for subgraph TrafficShapingRules
+	defaultRequestRule := GlobalSubgraphRequestRule{
```

Comment:
This would be the second place where we defined these defaults. I'd suggest fallback to the default in `NewTransportRequestOptions` and reuse it. In that case, it works in the tests and code is more coherent.

Example

```go
func NewTransportRequestOptions(cfg config.GlobalSubgraphRequestRule) TransportRequestOptions {
	return TransportRequestOptions{
		RequestTimeout:        or(cfg.RequestTimeout, 60* time.Second),
        // ...
	}
}
```